python/cugraph/cugraph/tests/centrality/test_edge_betweenness_centrality.py

```diff
@@ -336,32 +344,29 @@ def test_edge_betweenness_centrality(
         normalized=normalized,
         k=subset_size,
         weight=weight,
-        seed=subset_seed,
+        seed=42,
         result_dtype=result_dtype,
         edgevals=edgevals,
     )
     compare_scores(sorted_df, first_key="cu_bc", second_key="ref_bc")
 
 
-# FIXME: Fails for directed = False(bc score twice as much) and normalized = True.
 @pytest.mark.sg
 @pytest.mark.parametrize("graph_file", DATASETS_SMALL)
-@pytest.mark.parametrize("directed", [DIRECTED_GRAPH_OPTIONS[1]])
+@pytest.mark.parametrize("directed", DIRECTED_GRAPH_OPTIONS)
 @pytest.mark.parametrize("subset_size", [None])
-@pytest.mark.parametrize("normalized", [NORMALIZED_OPTIONS[0]])
+@pytest.mark.parametrize("normalized", NORMALIZED_OPTIONS)
 @pytest.mark.parametrize("weight", [None])
-@pytest.mark.parametrize("subset_seed", SUBSET_SEED_OPTIONS)
 @pytest.mark.parametrize("result_dtype", RESULT_DTYPE_OPTIONS)
 @pytest.mark.parametrize("use_k_full", [True])
 @pytest.mark.parametrize("edgevals", WEIGHTED_GRAPH_OPTIONS)
-# @pytest.mark.skip(reason="Skipping large tests")
+@pytest.mark.skip(reason="Skipping large tests")
```

Comment:
I noticed several of these tests are being skipped now, with the reason being they're large tests. Are they too large even with the small datasets being used here?

Reply:
yeah I think if they aren't supposed to be skipped then the `skip` decorators should definitely be removed.